### PR TITLE
chore: update storybook theming

### PR DESCRIPTION
**Pull request by nataliepina**
Breaking out theming changes into a smaller pull request. 

## Testing

Spin it up locally and view theme changes.

## Screenshots
Screen Shot 2021-08-30 at 7 50 46 PM

## Checklist for PR creator

- [ ] If any new components were added, there are exported from `packages/index.ts`
- [ ] If this PR is associated with a JIRA, it is mentioned in commit message footer ("Closes …")
- [ ] If this PR contains breaking changes, is stated in commit message body ("BREAKING CHANGE: …")
- [x] Info for applicable sections above is provided

### CODE REVIEW

File: .storybook/theme.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { create } from "@storybook/theming";
+import logo from "./static/logo.svg";
```
Comment (vacas5) [inline .storybook/theme.js:2]: did you need your webpack changes to support this?

Comment (nataliepina) [reply .storybook/theme.js:2]: I am thinking not (?) I am perplexed by this compared to my other branch where I had lots of logo rendering troubleshooting that did involve the webpack changes. I tried running this before adding any of that and the logo renders 🤷‍♀️

**Review by clintonmedbery — APPROVED** [review]
LGTM!

**Review by vacas5 — APPROVED** [review]
cool lets merge without then and see if we can avoid it altogether

**Pull request merged**